.github/workflows/build-and-release.yaml

```diff
@@ -0,0 +1,107 @@
+# SPDX-FileCopyrightText: 2025 Info
```

Comment:
I think if we are switching to using Info instead of Lifely in our SPDX headers we should make a conscious decision about this first. Also I would not use `Info` because nobody will ever find us that way. I would use `INFO.nl`.